testsuite-shading/pom.xml

```diff
@@ -59,8 +58,132 @@
           <skip>true</skip>
         </configuration>
       </plugin>
```

Comment:
These changes are because this deploy-plugin configuration doesn't work as or do what we expected?

Reply:
Yes it does not work as we use the nexus staging plugin while releasing which not supports what we need